Src/CSharpier/DocPrinter.cs

```diff
@@ -71,53 +77,53 @@ private Indent MakeIndent(Indent indent, Options options)
                 }
             }
 
-            flushSpaces();
+            FlushSpaces();
 
-            void addTabs(int count)
+            void AddTabs(int count)
             {
                 value += new string('\t', count);
                 length += options.TabWidth * count;
             }
 
-            void addSpaces(int count)
+            void AddSpaces(int count)
             {
                 value += new string(' ', count);
```

Comment:
Also, you should probably a `StringBuilder()` rather than string, since all you do are appends.
Also, I think it'll probably be faster to `Append(char)` in a loop than allocate a new string of the right length:
https://www.meziantou.net/stringbuilder-performance-pitfalls.htm#use-append-char-inst

Reply:
Looks like StringBuilder actually has .Append(' ', count)